storage/tsdb/promtsdb.go

```diff
@@ -100,28 +100,34 @@ func (promQuery *V3ioPromQuerier) Select(params *storage.SelectParams, oms ...*l
 	}
 
 	promQuery.logger.Debug("SelectParams: %+v", params)
+	overTimeSuffix := "_over_time"
 
 	if params.Func != "" {
 		// only pass xx_over_time functions (just the xx part)
 		// TODO: support count/stdxx, require changes in Prometheus: promql/functions.go, not calc aggregate twice
-		if strings.HasSuffix(params.Func, "_over_time") {
-			f := params.Func[0:3]
-			if params.Step == 0 && (f == "min" || f == "max" || f == "sum" || f == "avg") {
-				functions = f
+		if strings.HasSuffix(params.Func, overTimeSuffix) {
+			if promQuery.UseAggregates && promQuery.UseAggregatesConfig {
+				functions = strings.TrimSuffix(params.Func, overTimeSuffix)
 			} else {
-				noAggr = true
+				f := params.Func[0:3]
+				if params.Step == 0 && (f == "min" || f == "max" || f == "sum" || f == "avg") {
```

Comment:
Not really safe... Could be called `summary`, and that starts with `sum`... But not a regression in this PR.